packer-plugin-sdk/communicator/config.go

```diff
@@ -24,8 +24,9 @@ import (
 	"golang.org/x/crypto/ssh/agent"
 )
 
-// Config is the common configuration that communicators allow within
-// a builder.
+// Config is the common a builder uses to define and configure a Packer
```

Comment:
```suggestion
// Config is the common configuration a builder uses to define and configure a Packer
```
I missed the `configuration` here